### ISSUE TITLE
Pre-release version is ignoring the defined `preid`

### ISSUE THREAD

**Issue by cmario92** [phips28/gh-action-bump-version#104]
I am trying to control the version bump for all MAJOR, MINOR, PATCH and pre-release as per semver.

I have the following configuration:
![image](https://user-images.githubusercontent.com/5765285/123975202-d75e5380-d9c5-11eb-8774-9f09a0dab810.png)

I want every push to trigger the pre-release version bump, except those that include the wordings as above,
but I get `undefined` on my project version, for commit messages that do not include any wordings(pre-release bump expected):

```
messages: [ 'dependencies\nundefined' ]
currentBranch: main
current: 0.0.41-undefined.0 / version: prerelease --preid=undefined
current: 0.0.41-undefined.0 / version: prerelease --preid=undefined
new version: v0.0.41-undefined.1

✔  success   Version bumped!
```

Edit: my commit message for the POC was `dependencies`

**Comment by phips28** [comment]
can you use `default: 'prerelease'` with the upticks? just a first guess.

**Comment by cmario92** [comment]
this is what I tried:
![image](https://user-images.githubusercontent.com/5765285/123979835-bb5cb100-d9c9-11eb-8999-351342d8c3e8.png)

And the result was(commit message was `update`:
```
messages: [ 'updates\nundefined', 'update\nundefined' ]
✔  success   No version keywords found, skipping bump.
```

**Comment by phips28** [comment]
ok thats odd :/ I am not using all these features 🙈 those are from the community. so I am not 100% aware of these LOC and have not enough time right now to debug.
maybe you can debug this in the code directly.

**Comment by cmario92** [comment]
hmm, neither this configuration is working for me - I end up with `x.x.x-undefined.0` as a version:

![image](https://user-images.githubusercontent.com/5765285/123981044-b0565080-d9ca-11eb-96c9-eba3b7b56d6a.png)

I will try to get rid of the major/minor/patch wordings and see what is the outcome

**Comment by cmario92** [comment]
resolved with #106
